fakeserver/server_test.go

```diff
@@ -132,8 +208,6 @@ func TestCors(t *testing.T) {
 }
 
 func TestPersistAssignment(t *testing.T) {
-	os.Remove("testdata/assignments.yml")
```

Comment:
as far as i can tell this wasn't doing anything.

also, i've made sure the tests are written such that they won't have mystery guests

Reply:
Right I think we switched to a tmp dir approach for each test run, but these lines might've snuck in from the prior implementation:
https://github.com/Betterment/testtrack-cli/pull/50#discussion_r382941214

Reply:
ah yes. thanks for the sleuthing 🕵🏻